sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -1177,4 +1177,110 @@
       <Count>1000</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_observingMode</EFDB_Topic>
+    <Description>Information about the current observing mode.</Description>
+    <item>
+      <EFDB_Name>modeId</EFDB_Name>
+      <Description>Current observing mode id.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>mode</EFDB_Name>
+      <Description>Current observing mode as a human-readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>typeValue</EFDB_Name>
+      <Description>Id of the type of observation.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>type</EFDB_Name>
+      <Description>Type of observation as a human-readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_blockStatus</EFDB_Topic>
+    <Description>Status of an individual block.</Description>
+    <item>
+      <EFDB_Name>id</EFDB_Name>
+      <Description>Id of the block (e.g. OBS-123).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>statusId</EFDB_Name>
+      <Description>Id of the status.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>status</EFDB_Name>
+      <Description>Status of the block, as a human readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>iterationsCompleted</EFDB_Name>
+      <Description>Number of completed iterations.</Description>
```

Comment:
A block gets executed...
should this be `executionsCompleted` ?
Then the descriptions can be 
`Number of successful executions"  ?